Makefile

```diff
@@ -55,3 +55,30 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+BENCH_PATH ?= bench
+
+.PHONY: bench-deps
+bench-deps:
+	rm -rf ${BENCH_PATH}
+	mkdir ${BENCH_PATH}
+	go clean -testcache
+	cd ${BENCH_PATH} && git clone https://go.googlesource.com/perf && cd perf && go install ./cmd/benchstat
+	rm -rf ${BENCH_PATH}/perf
+
+.PHONY: bench
+bench:
+	go test -bench=. -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 | tee ${BENCH_PATH}/bench.txt
```

Comment:
Please put a path (current value is `.`) to a variable. And I believe we should search benchmarks recursively (`./...`)

Reply:
Not fixed now.
We should put here an absolute path and probably make it recursive.

Reply:
It is not clear what path is actually used for searching benchmark tests.
`go test ${TEST_PATH} -bench=. -run=^Benchmark -benchmem -benchtime=${DURATION} -count=${COUNT} `
first path is passed to `go test` and second one to option `-bench`.

Reply:
The `-bench` flag doesn't accept paths, only regexp.

Reply:
Adding the variable `BENCH_OPTIONS` seems to fix that mess.